[
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/526301901",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/1726#issuecomment-526301901",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/1726",
        "id": 526301901,
        "node_id": "MDEyOklzc3VlQ29tbWVudDUyNjMwMTkwMQ==",
        "user": {
            "login": "dashesy",
            "id": 873905,
            "node_id": "MDQ6VXNlcjg3MzkwNQ==",
            "avatar_url": "https://avatars.githubusercontent.com/u/873905?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/dashesy",
            "html_url": "https://github.com/dashesy",
            "followers_url": "https://api.github.com/users/dashesy/followers",
            "following_url": "https://api.github.com/users/dashesy/following{/other_user}",
            "gists_url": "https://api.github.com/users/dashesy/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/dashesy/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/dashesy/subscriptions",
            "organizations_url": "https://api.github.com/users/dashesy/orgs",
            "repos_url": "https://api.github.com/users/dashesy/repos",
            "events_url": "https://api.github.com/users/dashesy/events{/privacy}",
            "received_events_url": "https://api.github.com/users/dashesy/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2019-08-29T18:15:57Z",
        "updated_at": "2019-08-29T18:15:57Z",
        "author_association": "NONE",
        "body": "This is the start of graph when exporting, clearly it is Byte:\r\n\r\n```text\r\ngraph(%data : Byte(*, *, *, *)\r\n```",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/526301901/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    },
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/526319406",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/1726#issuecomment-526319406",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/1726",
        "id": 526319406,
        "node_id": "MDEyOklzc3VlQ29tbWVudDUyNjMxOTQwNg==",
        "user": {
            "login": "dashesy",
            "id": 873905,
            "node_id": "MDQ6VXNlcjg3MzkwNQ==",
            "avatar_url": "https://avatars.githubusercontent.com/u/873905?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/dashesy",
            "html_url": "https://github.com/dashesy",
            "followers_url": "https://api.github.com/users/dashesy/followers",
            "following_url": "https://api.github.com/users/dashesy/following{/other_user}",
            "gists_url": "https://api.github.com/users/dashesy/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/dashesy/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/dashesy/subscriptions",
            "organizations_url": "https://api.github.com/users/dashesy/orgs",
            "repos_url": "https://api.github.com/users/dashesy/repos",
            "events_url": "https://api.github.com/users/dashesy/events{/privacy}",
            "received_events_url": "https://api.github.com/users/dashesy/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2019-08-29T19:03:36Z",
        "updated_at": "2019-08-29T19:04:07Z",
        "author_association": "NONE",
        "body": "When I try ORT 0.5 I get slightly better error:\r\n\r\n> RuntimeError: [ONNXRuntimeError] : 1 : GENERAL ERROR : Load model from output/gr/hybrid_model_v2/tagger.onnx failed:This is an invalid model. Model input (data) does not have type information.\r\n\r\n\r\nBut that means PyTorch 1.2 exporter has exported an invalid model, with an input that does not have type information!\r\nThis was fine in previous versions of PyTorch onnx exporter\r\n\r\n",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/526319406/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    },
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/526331735",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/1726#issuecomment-526331735",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/1726",
        "id": 526331735,
        "node_id": "MDEyOklzc3VlQ29tbWVudDUyNjMzMTczNQ==",
        "user": {
            "login": "dashesy",
            "id": 873905,
            "node_id": "MDQ6VXNlcjg3MzkwNQ==",
            "avatar_url": "https://avatars.githubusercontent.com/u/873905?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/dashesy",
            "html_url": "https://github.com/dashesy",
            "followers_url": "https://api.github.com/users/dashesy/followers",
            "following_url": "https://api.github.com/users/dashesy/following{/other_user}",
            "gists_url": "https://api.github.com/users/dashesy/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/dashesy/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/dashesy/subscriptions",
            "organizations_url": "https://api.github.com/users/dashesy/orgs",
            "repos_url": "https://api.github.com/users/dashesy/repos",
            "events_url": "https://api.github.com/users/dashesy/events{/privacy}",
            "received_events_url": "https://api.github.com/users/dashesy/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2019-08-29T19:40:44Z",
        "updated_at": "2019-08-29T19:40:44Z",
        "author_association": "NONE",
        "body": "This is certainly a regression. Model exported with PyTorch 1.0 works and has correct type, but both PyTorch 1.1 and 1.2 export without type information and loading the model generated by 1.1 segfaults ORT",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/526331735/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    },
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/530022982",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/1726#issuecomment-530022982",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/1726",
        "id": 530022982,
        "node_id": "MDEyOklzc3VlQ29tbWVudDUzMDAyMjk4Mg==",
        "user": {
            "login": "faxu",
            "id": 20780999,
            "node_id": "MDQ6VXNlcjIwNzgwOTk5",
            "avatar_url": "https://avatars.githubusercontent.com/u/20780999?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/faxu",
            "html_url": "https://github.com/faxu",
            "followers_url": "https://api.github.com/users/faxu/followers",
            "following_url": "https://api.github.com/users/faxu/following{/other_user}",
            "gists_url": "https://api.github.com/users/faxu/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/faxu/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/faxu/subscriptions",
            "organizations_url": "https://api.github.com/users/faxu/orgs",
            "repos_url": "https://api.github.com/users/faxu/repos",
            "events_url": "https://api.github.com/users/faxu/events{/privacy}",
            "received_events_url": "https://api.github.com/users/faxu/received_events",
            "type": "User",
            "site_admin": true
        },
        "created_at": "2019-09-10T16:45:14Z",
        "updated_at": "2019-09-10T16:45:14Z",
        "author_association": "MEMBER",
        "body": "@spandantiwari @BowenBao  can you guys take a look at whether this is a pytorch issue?",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/530022982/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    },
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/545695872",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/1726#issuecomment-545695872",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/1726",
        "id": 545695872,
        "node_id": "MDEyOklzc3VlQ29tbWVudDU0NTY5NTg3Mg==",
        "user": {
            "login": "dashesy",
            "id": 873905,
            "node_id": "MDQ6VXNlcjg3MzkwNQ==",
            "avatar_url": "https://avatars.githubusercontent.com/u/873905?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/dashesy",
            "html_url": "https://github.com/dashesy",
            "followers_url": "https://api.github.com/users/dashesy/followers",
            "following_url": "https://api.github.com/users/dashesy/following{/other_user}",
            "gists_url": "https://api.github.com/users/dashesy/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/dashesy/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/dashesy/subscriptions",
            "organizations_url": "https://api.github.com/users/dashesy/orgs",
            "repos_url": "https://api.github.com/users/dashesy/repos",
            "events_url": "https://api.github.com/users/dashesy/events{/privacy}",
            "received_events_url": "https://api.github.com/users/dashesy/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2019-10-24T00:58:18Z",
        "updated_at": "2019-10-24T00:58:18Z",
        "author_association": "NONE",
        "body": "This is still an issue\r\n\r\n```text\r\n----> 1 sess = rt.InferenceSession(onnxfile)\r\n      2 t0 = time.time()\r\n      3 pred = sess.run(targets, {'data': blob})[0]\r\n      4 print(time.time() - t0)\r\n\r\n~/miniconda3/envs/py3/lib/python3.6/site-packages/onnxruntime/capi/session.py in __init__(self, path_or_bytes, sess_options)\r\n     27\r\n     28         if isinstance(path_or_bytes, str):\r\n---> 29             self._sess.load_model(path_or_bytes)\r\n     30         elif isinstance(path_or_bytes, bytes):\r\n     31             self._sess.read_bytes(path_or_bytes)\r\n\r\nRuntimeError: [ONNXRuntimeError] : 1 : GENERAL ERROR : Load model from /mnt/output/gr/tagging/Tagging12K/Tagging12K.onnx failed:This is an invalid model. Model input (data) does not have type information.\r\n```\r\n\r\nI cannot export models that I used to be able to export",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/545695872/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    }
]